system/mp/mpcomm.cpp

```diff
@@ -1932,16 +1937,29 @@ int CMPConnectThread::run()
         if (sock) {
             try {
                 sock->set_keep_alive(true);
-                size32_t rd;
+                size32_t rd = 0;
```

Comment:
debatable, but think cleaner if method that's about to fill scaler reference like this, ensures always set, i.e. clears to 0 at start of traceSlowReadTms()